{{cookiecutter.project_slug}}/webpack.config.js

```diff
@@ -85,37 +86,30 @@ module.exports = {
       filename: '[name].css',
     }),
     new SpriteLoaderPlugin(),
+    new CleanWebpackPlugin({
+      cleanOnceBeforeBuildPatterns: ['**/*', '!VERSION'],
+    }),
   ],
   devServer: {
     proxy: {
       '**': {
-        target: process.env.BACKEND_URL || "http://backend:8000",
+        target: process.env.BACKEND_URL || 'http://backend:8000',
       },
     },
-    allowedHosts: (process.env.ALLOWED_HOSTS || "localhost").split(/\s+/),
+    allowedHosts: (process.env.ALLOWED_HOSTS || 'localhost').split(/\s+/),
     host: '0.0.0.0',
     port: 3000,
+    // Enable `sockPort` if you use Pontsun configuration
+    // sockPort: 443,
     compress: true,
-    // Polling is required inside Vagrant boxes
-    watchOptions: {
-      poll: true,
-    },
     overlay: true,
-    // Here you can specify folders that contain your views
-    // So they’ll trigger a page reload when you change them
-    // contentBase: ['./app/views'],
-    // watchContentBase: true,
+    contentBase: [
+      './{{ cookiecutter.project_slug }}/**/templates/*.html',
+      './{{ cookiecutter.project_slug }}/**/templates/cms/**/*.html',
```

Comment:
I remember ContentBase not supporting globs and requiring to specify every single folder containing `html` files to work properly. Did that changed recently?

Reply:
Actually, I didn't know that and I see nothing in the [docs](https://webpack.js.org/configuration/dev-server/#devservercontentbase) but maybe I missed something. I just tried it out and

Reply:
I did not know that after looking at the [documentation](https://webpack.js.org/configuration/dev-server/#devservercontentbase) I don't see anything about that topic but I missed something maybe. Actually, I just tried it out and it seems to work.

Reply:
After some tests, it seems to work now. The glob I'd recommend is: `'./{{ cookiecutter.project_slug }}/**/templates/**/*.html'`